functionary/ui/templates/partials/modal.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal is-active" _="on closeModal remove me">
```

Comment:
The base.html has blocks for title and content, I'm not sure if you can get away with removing the modal_ from the suggested block names.

Reply:
Fixed